app/data/models/payment-request.js

```diff
@@ -0,0 +1,38 @@
+module.exports = (sequelize, DataTypes) => {
+  const paymentRequest = sequelize.define('paymentRequest', {
+    paymentRequestId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+    agreementNumber: DataTypes.STRING,
+    contractNumber: DataTypes.STRING,
+    correlationId: DataTypes.STRING,
+    currency: DataTypes.STRING,
+    deliveryBody: DataTypes.STRING,
+    dueDate: DataTypes.STRING,
+    invoiceNumber: DataTypes.STRING,
+    marketingYear: DataTypes.INTEGER,
+    referenceId: DataTypes.UUID,
+    schedule: DataTypes.STRING,
+    sitiAgriInvoiceNumber: DataTypes.STRING,
```

Comment:
Is the intention here that as we save each payment request, we determine what the Siti Agri invoice number would have been for those schemes that original from Siti Agri?

Do you think it would be better to have a separate table with the invoice number relationships?  eg `originalInvoiceNumber` with many `invoiceNumbers`?

Or do you think it would be better to handle any mapping at runtime in the application when working with the data?